app/mailers/claim_mailer.rb

```diff
@@ -215,13 +211,25 @@ def template_ids(claim)
   end
 
   def send_mail(template_id, personalisation)
-    template_mail(
-      template_id,
-      to: @claim.email_address,
-      subject: @subject,
-      reply_to_id: @policy.notify_reply_to_id,
-      personalisation:
-    )
+    if Rails.env.development?
+      Rails.logger.info(
+        [
+          "\n",
+          "Sending email to: #{@claim.email_address}",
+          "Template ID: #{template_id}",
+          "Subject: #{@subject}",
+          "Personalisation: \n#{personalisation.pretty_inspect}"
+        ].join("\n")
+      )
+    else
+      template_mail(
+        template_id,
+        to: @claim.email_address,
+        subject: @subject,
+        reply_to_id: @policy.notify_reply_to_id,
+        personalisation:
+      )
+    end
```

Comment:
this will actually prevent the email from sending though, which is useful for debugging. do we not want to do this on top of the default behaviour?

Reply:
I've never used that but will let it send emails through